src/cpp/server/residency/platform/durable_file_adapter_windows.cpp

```
@@ -0,0 +1,1171 @@
+    }
+
+private:
+    std::filesystem::path directory_;
+    HANDLE directory_handle_ = INVALID_HANDLE_VALUE;
+    HANDLE lock_handle_ = INVALID_HANDLE_VALUE;
+};
+
+} // namespace
+
+std::unique_ptr<DurableFileAdapter>
+make_platform_durable_file_adapter(const std::filesystem::path &directory) {
```

Cppcheck (2.21.0)

[style] The function 'make_platform_durable_file_adapter' is never used.

(unusedFunction)